documentation/course/e2/README.md

```diff
@@ -1147,7 +1272,7 @@ TODO @piterek130: opis zmian
 
 ### Logistyka
 
-TODO @mlodybercik: opis zmian
+W celu zwiększenia wygody użytkowników, dodano atrybut `ordered` do relacji między `Stop` a `Line`, który przechowuje informację o kolejności przystanków na danej linii. Wartość ta jest unikalna dla każdego przystanku na danej linii.
```

Comment:
It isn't unique in your db diagram

Reply:
But it won't be shown on the diagram. `order` is not unique by itself. `(order, line_id, stop_id)` tuple is.

Reply:
Look at line 1676

Reply:
Okay, that make sense